ObservableWebView/ContentView.swift

```diff
@@ -9,14 +9,18 @@ import SwiftUI
 
 struct ContentView: View {
   @State var webViewManager = ObservableWebViewManager()
-  
   @State private var webContentThemeColor: Color = .clear
+  @State private var toolbarStringText = ""
   
   var body: some View {
     VStack {
       
       ObservableWebView(manager: webViewManager)
         .frame(maxWidth: .infinity, maxHeight: .infinity)
+        .onAppear {
+          webViewManager.webView.allowsBackForwardNavigationGestures = true
+          webViewManager.load("https://duckduckgo.com")
```

Comment:
Due to ObservableWebViewManager's initializer code, an initial URL load will result in `canGoBack == true`.

Modify current implementation to have optional `urlString`:

```swift
extension ObservableWebViewManager {
  private struct Constants {
    static let aboutBlank = "about:blank"
    static let aboutBlankUrl = URL(string: Constants.aboutBlank)!
  }
}

@Observable
class ObservableWebViewManager {
  var webView: WKWebView = WKWebView()
  
  var urlString: String = ""

  init(urlString: String = Constants.aboutBlank) {
    self.webView = WKWebView()
    self.urlString = urlString
    load(urlString)
  }

  // ...

}
```

Reply:
Resolved: https://github.com/buzsh/ObservableWebView/pull/1/commits/9d21d0a7b45b6e4e01d96966883949636d9f3918